database/factories/PredefinedFilterFactory.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Company;
+use App\Models\User;
+use App\Models\Category;
+use App\Models\PredefinedFilter;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\PredefinedFilter>
+ */
+class PredefinedFilterFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+
+
+    public function definition(): array
+    {
+        return [
+            'name' => $this->faker->name,
+            'created_by' => User::factory(),
+            'filter_data' => ['company_id' => [1]],
```

Comment:
`1` shouldn't be hard coded since we might not have a company with an id of `1` in a test that doesn't create a company first.